roles/openshift_hosted/tasks/create_groups.yml

```diff
@@ -0,0 +1,15 @@
+---
```

Comment:
What is going on with this file?  I think you accidentally commit some local testing changes.